pkg/topology/pslice/pslice.go

```diff
@@ -95,6 +95,28 @@ func (s *PSlice) EachBinRev(pf topology.EachPeerFunc) error {
 	return nil
 }
 
+func (s *PSlice) BinPeers(bin uint8) []swarm.Address {
```

Comment:
r: if you remove the type from `swarm.MaxBin` constant as recommended above, then the `bin` can be simply an integer and the casting on `L:102` wouldn't be necessary